bucket/iddsampledriver-ge9-np.json

```
@@ -0,0 +1,53 @@
+{
+    "version": "0.0.1.2",
+    "description": "Windows Indirect Display Driver using the IddCx class extension driver. Virtual monitors for Windows 10/11.",
+    "homepage": "https://github.com/ge9/IddSampleDriver",
+    "license": "MIT,CC0 | Public Domain",
+    "notes": "Use \"C:\\IddSampleDriver\\option.txt\" to set the number of monitors and resolutions, then restart the driver from the Device Manager.",
+    "depends": "extras/nefcon",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ge9/IddSampleDriver/releases/download/0.0.1.2/IddSampleDriver.zip",
+            "hash": "c73d6f3b8af35d68b369de68200b1d4ad2a958d7c45d672a20a1ffe01d24f72c"
+        }
+    },
+    "extract_dir": "IddSampleDriver",
+    "pre_install": [
+        "if (-not $global) {",
+        "    error \"$app should be installed globally.\"",
+        "    break",
+        "}"
+    ],
+    "installer": {
+        "script": [
+            "# Remove unnecessary files",
+            "Remove-Item \"$dir\\certmgr.exe\"",
+            "Remove-Item \"$dir\\installCert.bat\"",
+            "",
+            "# Install certificates",
+            "certutil -addstore -f root \"$dir\\IddSampleDriver.cer\" | Out-Null",
+            "certutil -addstore -f TrustedPublisher \"$dir\\IddSampleDriver.cer\" | Out-Null",
+            "",
+            "# Copy option.txt to C:\\IddSampleDriver if it does not exist",
+            "if (!(Test-Path \"C:\\IddSampleDriver\\option.txt\")) {",
+            "    Write-Host -Foreground Yellow \"'C:\\IddSampleDriver\\option.txt' not found. Creating...\"",
+            "    ensure 'C:\\IddSampleDriver' | Out-Null",
+            "    Copy-Item -Path \"$dir\\option.txt\" -Destination 'C:\\IddSampleDriver\\'",
+            "}",
+            "",
+            "# Install driver",
+            "nefconw --create-device-node --hardware-id ROOT\\iddsampledriver --class-name Display --class-guid 4d36e968-e325-11ce-bfc1-08002be10318",
+            "nefconw --install-driver --inf-path \"$dir\\IddSampleDriver.inf\""
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "if ($cmd -ne 'uninstall') { return }",
+            "nefconw --remove-device-node --hardware-id ROOT\\iddsampledriver --class-guid 4d36e968-e325-11ce-bfc1-08002be10318"
```

```suggestion
            "nefconw --remove-device-node --hardware-id ROOT\\iddsampledriver --class-guid 4d36e968-e325-11ce-bfc1-08002be10318",
            "if ($purge) {",
            "    Write-Host -Foreground Yellow \"Removing 'C:\\IddSampleDriver'\"",
            "    Remove-Item 'C:\\IddSampleDriver' -Force -Recurse",
            "}"
```

This allows users to remove config when uninstalling the app via `-purge` or `-p` switch.
In packages using **persist**, `-purge` will remove all the persisted files.